command/ca/ca.go

```diff
@@ -165,7 +166,7 @@ func BetaCommand() cli.Command {
 		Description: `**step beta ca** enables beta access to new step-ca APIs. These
 commands may change, disappear, or be promoted to a different subcommand in the future.`,
 		Subcommands: cli.Commands{
-			admin.Command(),
+			adminbeta.Command(),
```

Comment:
So, to me, the question is what difference does it make when I plan to just complete remove these directories with the next release. It's more work to point to the new commands from `adminbeta` - and it's completely unnecessary because we will literally remove the files completely.

With `provisionerbeta` I'd rather not point to the new commands because the new commands sometimes use ca.json (if it exists), and that could actually break something for a customer. The old commands always use an API client.